lm_eval/tasks/mmlu_pro/default/_default_template_yaml

```diff
@@ -0,0 +1,15 @@
+dataset_path: sjyuxyz/MMLU-Pro-with-subset
```

Comment:
Is this OK to use there some other set than https://huggingface.co/datasets/TIGER-Lab/MMLU-Pro? Or am I missing something?

Reply:
Problem with the original dataset is that the categories are not stored as subsets. Instead, there is a 'category' feature. This goes against the mmlu dataset convention and thus cannot integrate well with the existing codebase.

The current set contains no further modifications from the original set except that the categories are split into subsets.

Reply:
Hi @ysjprojects, unfortunately this dataset differs from the original one. I haven't check everything, but eg. the first example in `dev` was corrupted: question changed from `'The symmetric group $S_n$ has $\n\\factorial{n}$ elements (...)'` to `'The symmetric group $S_n$ has $\nactorial{n}$ elements (...)'`. Also, I've noticed that the number of samples is different: the original one has 12032 rows, while the 'all' variant in the one divided into subsets has 10823.
Do you think it would be possible to fix it?

Reply:
Hi, @marta-sd I have pushed some changes to fix the dataset. It should be all good now.